hardware/esp8266com/esp8266/libraries/ESP8266WiFi/src/WiFiClient.cpp

```diff
@@ -40,10 +40,15 @@ extern "C"
 #include "include/ClientContext.h"
 #include "c_types.h"
 
+
+#define NO_PORT_BIND
```

Comment:
i was not sure if you need it for some reasons, so i not delete the code.
i remove it.